### PR TITLE
Update Global Relay config-settings.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Changes for: https://github.com/mattermost/mattermost-server/pull/14994

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -3312,6 +3312,14 @@ The email address your Global Relay server monitors for incoming compliance expo
 | This feature's ``config.json`` setting is ``"EmailAddress": ""`` with string input.                                                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Global SMTP Server Timeout
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:3315]: @isacikgoz, I'm not sure whether my explanation is accurate so please correct as needed. :)

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -3312,6 +3312,14 @@ The email address your Global Relay server monitors for incoming compliance expo
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 0 with a maximum setting of 1800 milliseconds.
```
Comment (isacikgoz) [inline source/administration/config-settings.rst:3317]: Actually the default value for this configuration parameter is `1800` seconds, and it shouldn't be `0`. Also, there is no maximum value for this.

File: source/administration/config-settings.rst (hunk 1 above)
Comment (isacikgoz) [inline source/administration/config-settings.rst:3315]:
```suggestion
Global Relay SMTP Server Timeout
```

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -3312,6 +3312,14 @@ The email address your Global Relay server monitors for incoming compliance expo
 | This feature's ``config.json`` setting is ``"EmailAddress": ""`` with string input.                                                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Global Relay SMTP Server Timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 0 with a maximum setting of 1800 milliseconds.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:3317]:
```suggestion
The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 1800 seconds. There is no maximum value.
```

File: source/administration/config-settings.rst (hunk 2 above)
Comment (justinegeffen) [reply source/administration/config-settings.rst:3317]: Great! Is "milliseconds" in the first sentence correct?

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -3312,6 +3312,14 @@ The email address your Global Relay server monitors for incoming compliance expo
...
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"SMTPServerTimeout": ""`` with numerical input.                                                                          |
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:3320]:
```suggestion
| This feature's ``config.json`` setting is ``"SMTPServerTimeout": "1800"`` with numerical input.                                                                          |
```

Comment (justinegeffen) [reply source/administration/config-settings.rst:3320]: @isacikgoz, is this correct or should it be blank (or 0)?

File: source/administration/config-settings.rst (hunk 2 above)
Comment (isacikgoz) [reply source/administration/config-settings.rst:3317]: Mmm, It should be `seconds` as well.

File: source/administration/config-settings.rst (hunk 3 above)
Comment (isacikgoz) [reply source/administration/config-settings.rst]: I removed the maximum value sentence, there is no need for mentioning it :)

**Pull request merged**